configure.ac

```diff
@@ -49,7 +50,12 @@ elif test $HAVE_pacemaker = 1; then
     if test $HAVE_cmap = 0; then
         AC_MSG_NOTICE(No package 'cmap' found)
     else
-	CPPFLAGS="$CPPFLAGS $cmap_CFLAGS"
+        CPPFLAGS="$CPPFLAGS $cmap_CFLAGS"
+    fi
+    if test $HAVE_votequorum = 0; then
+        AC_MSG_NOTICE(No package 'votequorum' found)
```

Comment:
I would say "library" instead of "package", otherwise users may start looking for a distro package by that name